developer/js/tests/fixtures/example.qaa.trivial/model.ts

```diff
@@ -0,0 +1,5 @@
+const source: LexicalModelSource = {
+  format: 'trie-1.0',
+  sources: ['tests/fixtures/example.qaa.trivial/wordlist.tsv'],
```

Comment:
One last thing: this needs to be a path relative to model.ts, not to where the build is run from.